clients/java/src/main/java/io/camunda/zeebe/client/api/search/query/SearchRequestBuilders.java

```diff
@@ -13,18 +13,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.camunda.zeebe.client.api.search;
-
-import io.camunda.zeebe.client.impl.search.DecisionDefinitionFilterImpl;
-import io.camunda.zeebe.client.impl.search.DecisionDefinitionSortImpl;
-import io.camunda.zeebe.client.impl.search.DecisionRequirementsFilterImpl;
-import io.camunda.zeebe.client.impl.search.DecisionRequirementsSortImpl;
-import io.camunda.zeebe.client.impl.search.ProcessInstanceFilterImpl;
-import io.camunda.zeebe.client.impl.search.ProcessInstanceSortImpl;
-import io.camunda.zeebe.client.impl.search.SearchRequestPageImpl;
-import io.camunda.zeebe.client.impl.search.UserTaskFilterImpl;
-import io.camunda.zeebe.client.impl.search.UserTaskSortImpl;
-import io.camunda.zeebe.client.impl.search.VariableValueFilterImpl;
+package io.camunda.zeebe.client.api.search.query;
```

Comment:
🔧 This file shouldn't be in this package IMO. I would move it into the root of `/impl/search`